jac-uk/admin
# Dependency updates 13 Oct 2022

## What's included?
Update various dependencies

## Who should test?
✅ Product owner
✅ Developers

## How to test?
Perform a clean install from this branch and ensure the following work as expected:
 - `npm run lint`
 - `npm run test`
 - `npm run serve` 
 - Check admin is working in particular make sure the Doc and XLSX downloads are working

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
